app/_data/docs_nav_konnect.yml

```diff
@@ -134,17 +134,17 @@
     - text: Troubleshoot
       url: /dev-portal/troubleshoot
 
-- title: Analyze
-  icon: /assets/images/icons/konnect/icn-vitals-nav.svg
+- title: Analytics
+  icon: /assets/images/icons/konnect/icn-analytics-nav.svg
   items:
-    - text: Introduction to Vitals
-      url: /vitals/
+    - text: Introduction to Analytics
+      url: /analytics/
     - text: Overview Dashboard
-      url: /vitals/overview-dashboard/
+      url: /analytics/overview-dashboard/
     - text: Analyze Services and Routes
-      url: /vitals/analyze/
+      url: /analytics/analyze/
```

Comment:
Is it worth renaming this file so we don't have this awkward `analytics/analyze` URL? Maybe `analytics/services-and-routes`?

Reply:
done